src/Stateless/StateConfiguration.Async.cs

```diff
@@ -21,7 +21,7 @@ public partial class StateConfiguration
             /// <returns></returns>
             public StateConfiguration InternalTransitionAsync(TTrigger trigger, Func<Transition, Task> entryAction)
             {
-                if (entryAction == null) throw new ArgumentNullException(nameof(entryAction));
+                Enforce.ArgumentNotNull(entryAction, nameof(entryAction));
```

Comment:
I don't mind!
Actually when I first started using Stateless, I thought that the Enforce class was a bit unnecessary.
I'll remove the references to Enforce.